riot/riot.py

```diff
@@ -86,6 +86,15 @@ def version(self) -> str:
         version = output.decode().strip().split(" ")[1]
         return version
 
+    @functools.lru_cache()
+    def version_info(self) -> t.Tuple[int, int, int]:
+        version_head = "".join(
+            list(itertools.takewhile(lambda _: _.isdigit() or _ == ".", self.version()))
+        )
+        # Return (major, minor, patch)
+        version_info = tuple(int(_) for _ in version_head.split(".")) + (0, 0, 0)
+        return (version_info[0], version_info[1], version_info[2])
```

Comment:
```suggestion
        return version_info
```
?

Reply:
I had issues with mypy typing, since I explicitly want a `Tuple[int, int, int]`. If someone has another idea to make it work, I am game.

The type from L95 is `Tuple[int, ...]`